content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+choose-to-receive, encode-and-retrieve, and choose-to-transmit.
+Focusing on emotional selection studies showing that participants are more willing to pass on stimuli that elicit disgust [@heath_emotional_2001], the authors showed that any of those phases can be the target of a selection pressure.
+Such pressures can additionally counteract each other, showing that the transmission process is more complex than was initially assumed.
+Studies of digital media have adopted the increased level of detail, as digital communication lets users copy-paste and bypass the encode-and-retrieve phase of transmission [@acerbi_cultural_2016].
+
+Methodological exploration has also led to more free-form interaction setups:
+in a study of the transmission of risk perception, @moussaid_amplification_2015 chose to make the interaction and underlying transmission of information an open-ended process.
+After initiating a chain by providing the first participant with a set of documents to read on their own, later participants were left to talk freely in successive dyads, and the whole session was recorded on film for subsequent analysis.
+An analogous change of setup was operated by @caldwell_studying_2008 [-@caldwell_experimental_2008] who investigated the cumulative aspect of the evolution of building techniques in a series of experiments asking participants to construct spaghetti towers or paper planes, later evaluated by their height and flight distance.
+A crucial point in those setups, Sterelny notes [-@sterelny_cultural_2017, n. 12], is that participants could observe the preceding generation during their experimenting and building, meaning they had access to rich context for the learning phase of transmission.
+Taking another step to embrace interaction with a setup where participants had to repeatedly build Lego cars in pairs (though without transmission), @mcgraw_cultures_2014 take the complete joint interaction to be their object of study and propose to use the constructions resulting from interacting dyads as a trace of the processes that took place in the interaction.
+By looking at the characteristics of the cars built by participants, the authors claim to present "methods for discerning, and quantifying, schema-like intersubjective understandings in material form" [@mcgraw_cultures_2014, p. 4; see also @mitkidis_building_2015; and @wallot_beyond_2016], effectively integrating critiques that I discuss below in @sec:intro-review-criticisms.
+
+The works reviewed here show that there is much room for exploration, on one side, of the methodological choices in transmission chain experiments (some of which will turn out to be more important than others in the trends observed), and on the other, of the theoretical background that sustains a given study.
+Such experiments have therefore much to bring to underlying theoretical debates.
+
+
+#### Language change: Experimental semiotics and Iterated learning
```

Comment:
J'avais séparé parce que 1) effectivement c'est des communautés différentes à la base (même si ça converge comme noté en fin de 1.1.3.3), et 2) pour le langage il y a un aspect supplémentaire de "système de communication" (dont on étudie l'évolution) vs. "usage du système" (dont le détail influe sur l'évolution du système, mais qui n'importe pas plus que ça sinon).

TODO
- [ ] reduce numbering depth
- [ ] make explicit the reasons for separating here